internal/pkg/api/handleEnroll_test.go

```diff
@@ -90,5 +95,185 @@ func TestEnroll(t *testing.T) {
 	if resp.Action != "created" {
 		t.Fatal("enroll failed")
 	}
+}
+
+func TestEnrollerT_fetchStaticTokenPolicy(t *testing.T) {
+	cacheBuilder := func(policies ...model.Policy) func() cache.Cache {
+		return func() cache.Cache {
+			c, _ := cache.New(config.Cache{
+				NumCounters:  100,
+				MaxCost:      100000,
+				PolicyKeyTTL: time.Hour,
+			})
+			for _, p := range policies {
+				c.SetPolicy(p)
+			}
+			// sleep to allow cache to be populated through buffers
+			// since we cant do cache.Wait()
+			// https://github.com/dgraph-io/ristretto/blob/main/cache.go#L213
+			time.Sleep(100 * time.Millisecond)
+			return c
+		}
+	}
+	bulkerBuilder := func(policies ...model.Policy) func() bulk.Bulk {
+		return func() bulk.Bulk {
+			bulker := ftesting.NewMockBulk()
 
+			hits := []es.HitT{}
+			for _, p := range policies {
+				b, _ := json.Marshal(p)
+				hits = append(hits, es.HitT{
+					Source: b,
+				})
+			}
+			res := &es.ResultT{
+				HitsT: es.HitsT{},
+				Aggregations: map[string]es.Aggregation{
+					"policy_id": {
+						Buckets: []es.Bucket{
+							{
+								Aggregations: map[string]es.HitsT{
+									"revision_idx": {
+										Hits: hits,
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			bulker.On("Search", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(res, nil)
+			return bulker
+		}
+	}
+	type fields struct {
+		staticPolicyTokens config.StaticPolicyTokens
+		cache              func() cache.Cache
+		bulker             func() bulk.Bulk
+	}
+	type args struct {
+		enrollmentAPIKey *apikey.APIKey
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    *model.EnrollmentAPIKey
+		wantErr bool
+	}{
+		{
+			name: "disabled",
+			fields: fields{
+				staticPolicyTokens: config.StaticPolicyTokens{
+					Enabled: false,
+				},
+				cache:  cacheBuilder(),
+				bulker: bulkerBuilder(),
+			},
+			args: args{
+				enrollmentAPIKey: &apikey.APIKey{},
+			},
+			want:    nil,
+			wantErr: false,
+		},
+		{
+			name: "found in cache13",
```

Comment:
```suggestion
			name: "policy found in cache",
```
(nit) makes test name a little clearer